pkg/task/task.go

```diff
@@ -445,6 +459,28 @@ func (m *taskExecutor) asExtnV1B1Deploy() (*api_extn_v1beta1.Deployment, error)
 	return d.AsExtnV1B1Deployment()
 }
 
+// asCStorPool generates a CstorPool object
+// out of the embedded yaml
+func (m *taskExecutor) asCStorPool() (*v1alpha1.CStorPool, error) {
+	d, err := m_k8s.NewCStorPoolYml("CStorPool", m.runtask.TaskYml, m.templateValues)
+	if err != nil {
+		return nil, err
+	}
+
+	return d.AsCStorPoolYml()
+}
+
+// asStoragePool generates a StoragePool object
+// out of the embedded yaml
+func (m *taskExecutor) asStoragePool() (*v1alpha1.StoragePool, error) {
+	d, err := m_k8s.NewStoragePoolYml("StoragePool", m.runtask.TaskYml, m.templateValues)
```

Comment:
Can "StoragePool" be declared as a constant/variable ?

Reply:
I dont see any reason to do so.
IMO we declare some string as constant when we use it at a lot of places.
Doint this kind of activity excessively can bring the overhead of package import and all which
Sometimes lead to cyclic imports.
According to me,we should declare as constants when it is required.
Although open for discussion regarding this.